src/main/java/picard/util/IntervalListTools.java

```diff
@@ -490,6 +507,12 @@ protected int doWork() {
         if ((SECOND_INPUT != null && !SECOND_INPUT.isEmpty()) && !ACTION.takesSecondInput) {
             errorMsgs.add("SECOND_INPUT was provided but action " + ACTION + " doesn't take a second input.");
         }
+        if (COUNT_OUTPUT != null && OUTPUT_VALUE == Output.NONE) {
```

Comment:
should be in customCommandLineValidation

Reply:
I believe this is in `customCommanLineValidation`...

Reply:
not sure why github claims this is in `doWork` when viewing changes... if you view the full file it is in `customCommandLineValidation`.

Reply:
odd.